compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -93,6 +95,15 @@ static llvm::cl::opt<bool> clNormalizeInputIndexingMap(
     llvm::cl::desc("Enable normalizing input indexing map to identity"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clViewDispatchGraph(
+    "iree-flow-view-dispatch-graph",
```

Comment:
let's call this iree-flow-dump-dispatch-graph to match the other iree-*-dump-* passes we have.
'view' would be used if this opened up the dot viewer, and 'print' would be used if it was going to stdout, but since it's dumping it to a file 'dump' is better